lib/libvcc/vcc_backend_util.c

```diff
@@ -106,17 +106,17 @@ vcc_IsField(struct vcc *tl, struct token **t, struct fld_spec *fs)
 			fs->found = t_field;
 			return;
 		}
-		VSB_cat(tl->sb, "Field ");
+		vcc_Complain(tl, "Field ");
 		vcc_ErrToken(tl, t_field);
-		VSB_cat(tl->sb, " redefined at:\n");
+		vcc_Complain(tl, " redefined at:\n");
```

Comment:
I am not against it, but in general, I think we should handle refactoring (as introducing `vcc_Complain` and renaming `sb` -> `err_sb`) separate from such PRs.